app/src/test/java/org/mozilla/focus/onboarding/OnboardingInteractorTest.kt

```diff
@@ -3,35 +3,59 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 package org.mozilla.focus.onboarding
 
+import android.os.Build
 import androidx.preference.PreferenceManager
 import androidx.test.core.app.ApplicationProvider
 import org.junit.Assert
 import org.junit.Before
 import org.junit.Test
 import org.junit.runner.RunWith
 import org.mockito.Mock
+import org.mockito.Mockito.spy
+import org.mockito.Mockito.times
+import org.mockito.Mockito.verify
 import org.mockito.MockitoAnnotations
-import org.mozilla.focus.fragment.onboarding.OnboardingFragment
-import org.mozilla.focus.fragment.onboarding.OnboardingInteractor
+import org.mozilla.focus.fragment.onboarding.OnBoardingInteractor
+import org.mozilla.focus.fragment.onboarding.OnBoardingInteractor.Companion.ONBOARDING_PREF
 import org.mozilla.focus.state.AppStore
 import org.robolectric.RobolectricTestRunner
+import org.robolectric.annotation.Config
 
 @RunWith(RobolectricTestRunner::class)
 class OnboardingInteractorTest {
 
     @Mock
     private lateinit var appStore: AppStore
+    private lateinit var onBoardingInteractor: OnBoardingInteractor
 
     @Before
     fun init() {
         MockitoAnnotations.openMocks(this)
+        onBoardingInteractor = spy(
+            OnBoardingInteractor(
+                appStore,
+                ApplicationProvider.getApplicationContext(),
+                "1"
+            )
+        )
     }
 
     @Test
-    fun `given onboarding, when start browsing is pressed, then onboarding flag is true`() {
-        OnboardingInteractor(appStore).finishOnboarding(ApplicationProvider.getApplicationContext(), "1")
+    fun `given onBoarding, when start browsing is pressed, then onBoarding flag is true`() {
+        OnBoardingInteractor(
+            appStore,
+            ApplicationProvider.getApplicationContext(),
+            "1"
+        ).finishOnBoarding()
         val prefManager =
             PreferenceManager.getDefaultSharedPreferences(ApplicationProvider.getApplicationContext())
-        Assert.assertEquals(true, prefManager.getBoolean(OnboardingFragment.ONBOARDING_PREF, false))
+        Assert.assertEquals(true, prefManager.getBoolean(ONBOARDING_PREF, false))
```

Comment:
nit: `assertEquals` should be import to avoid the `Assert.assertEquals` call.
Also there should be a `newline` above this to separate test sections.

Reply:
Done